.cfignore

```diff
@@ -9,3 +9,8 @@ cg-deck
 cg-dashboard
 /screenshots/
 static/tests/
+.org.chromium*
+# only exclude these index.html files from the VNC service.
+# they only appear in the root folder.
+/index.html*
```

Comment:
I think a watch util would probably be best, then we don't have to run any commands. I found this [script](https://gist.github.com/senko/1154509), but it requires `inotify`, which I don't believe is available on a mac. I think the analog for OSX is `fsevents`. I also found this library on npm: https://www.npmjs.com/package/watchman. There is also `chokidar` (https://github.com/paulmillr/chokidar), which would probably do what we want. Its used by most of the build libraries (gulp, webpack, etc). It should be pretty easy to add a hook to listen for `index.html` file creation and then run something to delete them.